src/components/Home-Repos.js

```diff
@@ -111,6 +123,13 @@ Last Updated:
     this.props.onRepoClick(this.props.userData.repositories.edges[index].node);
   }
 
+  repoStarCount = (starCount) => {
+    if (starCount) {
```

Comment:
This should be `if (node.stargazers && node.stargazers.totalCount)`

...

`return node.stargazers.totalCount !== null ? node.stargazers.totalCount : '?'`